packages/orbit-components/src/Card/Card.stories.tsx

```diff
@@ -303,16 +295,7 @@ export const CardWithMixedSections: Story = {
         </ButtonLink>
       }
     >
-      <CardSection
-        expandable
-        title={sectionTitle}
-        actions={
-          <ButtonLink compact size="small" type="secondary">
-            Button
-          </ButtonLink>
-        }
-        description={sectionDescription}
-      >
+      <CardSection expandable title={sectionTitle} description={sectionDescription}>
```

Comment:
CardSection is a `button` element that includes `div`, which is not correct according to the [Phrasing content ](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories) rules.

Reply:
Changed to div with role button and also added keyboard functionality